sankeys/migrations/0002_sankeydata_delete_sankeyfinancialdata.py

```
@@ -0,0 +1,54 @@
+# Generated by Django 6.0.1 on 2026-01-20 04:48
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+# 데이터를 이전하기 위한 함수 정의
+def migrate_old_data(apps, schema_editor):
```

Ruff (0.14.13)

Unused function argument: `schema_editor`

(ARG001)